octobot_evaluators/evaluator/strategy_evaluator.py

```diff
@@ -51,7 +51,9 @@ async def start(self, bot_id: str) -> bool:
         """
         await super().start(bot_id)
         self.consumer_instance = await get_chan(MATRIX_CHANNEL, self.matrix_id).new_consumer(
-            self.strategy_matrix_callback, priority_level=self.priority_level)
+            self.strategy_matrix_callback,
+            priority_level=self.priority_level,
+            exchange_name=self.exchange_name)
```

Comment:
I don't remember if exchange can be None, If it's possible we should use `exchange_name=self.exchange_name if self.exchange_name else CHANNEL_WILDCARD`.